### PR TITLE
Added player only mode

### PR DESCRIPTION
**Pull request by J3fftw1**
added player only mode for mob drops
and updated the dependencies

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,26 +30,26 @@
...
             <artifactId>dough-common</artifactId>
-            <version>1.0.3</version>
+            <version>1.1.2</version>
         </dependency>
 
         <dependency>
             <groupId>org.bstats</groupId>
             <artifactId>bstats-bukkit</artifactId>
-            <version>2.2.1</version>
+            <version>3.0.0</version>
```
Comment (WalshyDev) [inline pom.xml:52]: Major version change and pretty sure this has quite a few breaking changes. However, not seeing any changes relating to it so err. Does this actually compile?

File: src/main/java/dev/walshy/sfmobdrops/SfMobDrops.java [hunk 2]
```diff
@@ -84,6 +84,9 @@ protected void loadConfig() {
 
     @EventHandler
     public void onMobDeath(@Nonnull EntityDeathEvent e) {
+
```
Comment (WalshyDev) [inline src/main/java/dev/walshy/sfmobdrops/SfMobDrops.java:87]:
```suggestion
```

File: src/main/java/dev/walshy/sfmobdrops/SfMobDrops.java [hunk 3]
```diff
@@ -84,6 +84,9 @@ protected void loadConfig() {
...
+        if (getConfig().getBoolean("settings.only-player-drops", false) && e.getEntity().getKiller() == null) return;
```
Comment (WalshyDev) [inline src/main/java/dev/walshy/sfmobdrops/SfMobDrops.java:88]:
```suggestion
        if (getConfig().getBoolean("settings.only-player-drops", false) && e.getEntity().getKiller() == null) {
            return;
        }
```

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: src/main/java/dev/walshy/sfmobdrops/SfMobDrops.java (hunk 3 above)
Comment (WalshyDev) [inline src/main/java/dev/walshy/sfmobdrops/SfMobDrops.java:88]: This path doesn't even exist in the config.

You did not test this code...

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (J3fftw1) [reply pom.xml:52]: yea it actually compiles

File: src/main/java/dev/walshy/sfmobdrops/SfMobDrops.java (hunk 3 above)
Comment (J3fftw1) [reply src/main/java/dev/walshy/sfmobdrops/SfMobDrops.java]: whoops

**Pull request closed without merging**